wireless_modules/mqtt_test/boot.py

```diff
@@ -0,0 +1,27 @@
+from config import ESSID
+from config import PASSWORD
+
+def connect(eesid, password):
+    import network
+    sta_if = network.WLAN(network.STA_IF)
+    if not sta_if.isconnected():
+        print('connecting to network...')
+        sta_if.active(True)
+        sta_if.connect(eesid, password)
+        while not sta_if.isconnected():
+            pass
```

Comment:
What did you want this to do? A comment might help with what you intend for this whole loop to do.

Reply:
It just waits for the internet to connect before moving on.

Reply:
A comment would be handy as it is a bit confusing